gateway/ln-gateway/src/db.rs

```diff
@@ -65,6 +66,7 @@ pub struct GatewayConfiguration {
     pub num_route_hints: u32,
     #[serde(with = "serde_routing_fees")]
     pub routing_fees: RoutingFees,
+    pub network: Network,
```

Comment:
Does this need to be stored in the database?

Reply:
I believe so because gatewayd should reboot with the same network configuration, and check all federations are on the same network reestablishing connections